cfgen/cfgen

```diff
@@ -830,23 +830,42 @@ def oids_to_dhall(oids: List[Oid]) -> str:
 NetProtocol = Enum('NetProtocol', 'tcp o2ib')
 
 
+ProcT = Enum('ProcT', 'hax m0_server m0_client_s3 m0_client_other')
+ProcT.__doc__ = 'Type of process'
+
+
+class LnetPortalGroup(IntEnum):
```

Comment:
If we keep these values same as LibfabPortalGroup MSByte
i.e. 2,3,4,5 resp., then would be nice i think for quick understanding / guessing.

Reply:
That is okay, we have been following some conventions and neither of the ports are permanent, It is possible that libfabric ports may change in near future.